### PR TITLE
Versioneer

### PR DESCRIPTION
**Pull request by adybbroe**
Use versioneer to control versioning

 - [ ] Tests passed 
 - [x] Passes ``git diff origin/master **/*py | flake8 --diff``

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,16 @@ script: coverage run --source=trollsched setup.py test
 after_success:
 - if [[ $PYTHON_VERSION == 3.6 ]]; then coveralls; fi
 #
+deploy:
+  - provider: pypi
+    user: adybbroe
+    password:
+      secure: SY0qo7sZXDjDx0DHvuXrHvL9VTAulgU/T33d6UWXf469jT9DOexuZ2VYLgJbYQen5FSe5JmQE0ZMdId1cb8IPP/77qCgQK6f0lRDa43fSYXhcD+fHzlQskievJrwamkRYx6WBrJbwGAKBNinUgNSaTdbh9XUugziGFiOHUfVppM=
+    distributions: sdist bdist_wheel
```
Comment (djhoese) [inline .travis.yml]: I think you'll want to remove the wheel. I don't think it provides any benefit. If you really want the wheel I think you should look at adding the information to setup.cfg to say `universal=1` in a `bdist_wheel` section. That is, if this is a pure python package.

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+loghub pytroll/pytroll-schedule -u <username> -st v0.5.0 -plg bug "Bugs fixed" -plg enhancement "Features added" -plg documentation "Documentation changes"
+```
+
+Don't forget to commit!
+
+
+5. Create a tag with the new version number, starting with a 'v', eg:
+
+```
+git tag v0.5.1 -m "Version 0.5.1
```
Comment (djhoese) [inline RELEASING.md]: Missing quote at the end.

File: setup.cfg [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+# re-run 'versioneer.py setup' after changing this section, and commit the
+# resulting files.
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = trollsched/version.py
+versionfile_build = 
+tag_prefix = v
+parentdir_prefix =
```
Comment (djhoese) [inline setup.cfg]: If you run coverage (or flake8) you may want to cover the settings from satpy. They include telling coverage to ignore the versioneer files.

File: setup.py [hunk 4]
```diff
@@ -29,6 +29,7 @@
 from setuptools import setup
 import imp
 import sys
+import versioneer
 
 version = imp.load_source('trollsched.version', 'trollsched/version.py')
```
Comment (djhoese) [inline setup.py]: This line can be removed now, right?

File: RELEASING.md (hunk 2 above)
Comment (adybbroe) [reply RELEASING.md]: Yes, thanks, saw that too late

File: setup.py (hunk 4 above)
Comment (adybbroe) [reply setup.py]: Ahh, yeah, right

File: setup.cfg (hunk 3 above)
Comment (adybbroe) [reply setup.cfg]: Thanks.

File: .travis.yml (hunk 1 above)
Comment (adybbroe) [reply .travis.yml]: Thanks, done

File: setup.py (hunk 4 above)
Comment (djhoese) [reply setup.py]: Make sure you remove the `import imp` line too.

Comment (adybbroe) [reply setup.py]: Oh yes, thanks. Done

**Comment by adybbroe** [comment]
@djhoese I believe it is ready to be approved. Agree?

File: setup.cfg (hunk 3 above)
Comment (djhoese) [reply setup.cfg]: Doesn't look like you actually did this. Do you still want to add it?

Comment (adybbroe) [reply setup.cfg]: Strange, it is in, indeed

**Pull request merged**